public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx

```diff
@@ -66,19 +70,28 @@ export const dBHomeComponent = (
             this.state = {
                 surveyInfo: [],
                 domainInfo: [],
-                physicalMesurmentsInfo: []
+                physicalMesurmentsInfo: [],
+                searchWord: ''
             };
         }
 
+        search = _.debounce((val) => {
+            this.setState({searchWord: val});
+            this.getDomainInfos();
+            },
+        1000);
+
         // life cycle hook
         componentWillMount() {
             this.getDomainInfos();
         }
 
```

Comment:
Would also need to add a new function to handle the change from the child component, something like:

```
handleChange(val) {
  this.setState({searchWord: val});
  this.search(val);
}
```

Reply:
Changed this. Thanks.

Reply:
Changed this. Thanks.